src/main/java/org/example/controller/AthleteController.java

```diff
@@ -0,0 +1,26 @@
+package org.example.controller;
+
+import org.example.entity.Athlete;
+import org.example.mapper.AthleteJsonMapper;
+import org.example.service.AthleteService;
+
+
+public class AthleteController {
+    private final AthleteService athleteService;
+    private final AthleteJsonMapper athleteJsonMapper;
+
+    public AthleteController(AthleteService athleteService, AthleteJsonMapper athleteJsonMapper) {
```

Comment:
AthleteJsonMapper - It is Ok, but maybe you can inject just ObjectMapper everywhere ?  Create single ObjectMapper, set all modules and then just inject wherever you need mapping from/to json